### ISSUE TITLE
twisted.conch.ssh.channel should be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8649]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)| @einarfd reported|
|-|-|
|Trac ID|trac#8649|
|Type|enhancement|
|Created|2016-07-21 15:32:15Z|

Added a pull request for this bug at https://github.com/twisted/twisted/pull/385

<details><summary>Searchable metadata</summary>

```
trac-id__8649 8649
type__enhancement enhancement
reporter__einarfd einarfd
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469115135215865 1469115135215865
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)|@einarfd removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 5ed995e88fc771200923731b99e0c0b1531b8b02
```
#!CommitTicketReference repository="" revision="5ed995e88fc771200923731b99e0c0b1531b8b02"
Merge pull request #385 from einarfd/8649-einarfd-ssh-channel

Author: einarfd
Reviewer: rodrigc
Fixes: #8649 

Port twisted.conch.ssh.channel to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
